examples/Cargo.toml

```diff
@@ -10,7 +10,7 @@ futures = "0.3.6"
 openssl = "0.10.32"
 rustyline = "9"
 rustyline-derive = "0.6"
-scylla = {path = "../scylla", features = ["ssl", "cloud", "chrono", "time", "num-bigint-03", "num-bigint-04", "bigdecimal-04"]}
+scylla = {path = "../scylla", features = ["ssl", "cloud", "chrono-04", "time-03", "num-bigint-03", "num-bigint-04", "bigdecimal-04"]}
```

Comment:
It's not like both of these features cannot be used at the same time. The user might (for some weird reason) want to support (de)serialization for both versions of num_bigint::BigInt.

There is other problem, though - `BigInt` is not used in the examples at all... This means that we do not make use of any of these features anyway. I'm not sure why I added these features to examples. Probably for some reason related to vscode rust-analyzer. As I remember, the vscode' analyzer compiler the driver code with the features used in the examples/ by default. However, this can be configured in the workspace settings.

I think we could remove unused features from examples, but is it really necessary?

Reply:
If we really want to have all features enabled in examples, I'd suggest using "full_serialization" feature instead of listing them all manually.